sheets/management/commands/process_refund_requests.py

```diff
@@ -0,0 +1,33 @@
+"""
+Parses refund request row(s), reverses orders/enrollments, and updates the spreadsheet
+to reflect the processed request(s).
+"""
+from django.core.management import BaseCommand
+
+from mitol.google_sheets_refunds.api import RefundRequestHandler
+
+
+class Command(BaseCommand):
```

Comment:
This command should probably be in `ol-django`, but for the sake of getting this merged today let's go ahead with this as-is and refactor in a follow up.